times_reader/transforms.py

```diff
@@ -1253,7 +1253,7 @@ def name_comm_group(df):
             range="",
             filename="",
             uc_sets={},
-            tag="COMM_GROUPS",
+            tag="TOPOLOGY",
```

Comment:
If we've renamed the table to `TOPOLOGY`, would it make sense to also rename this function to `generate_topology`?

Reply:
I thought about that. :-) However we already have `process_topology`. In `generate_commodity_groups` we are basically expanding `ProcessTopology`. I guess, we could depreciate one of them, but probably best to create a seperate PR focusing on this revision. I wonder whether we should wait until we are more clear on the `times_model` class?